README.md

```diff
@@ -20,6 +20,16 @@ Here are some of the benefits of using this library:
 
 **API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
 
+## Index
+* [Feature Flags](#feature-flags)
+* [Feature Filters](#feature-filters)
+* [Registration](#registration)
+* [Consumption](#consumption)
+* [Targeting](#targeting)
+* [Caching](#caching)
+* [Custom Feature Providers](#custom-feature-providers)
+* [Contributing](#contributing)

```

Comment:
I would recommend leaving off this piece. It's not really relevant to referencing feature management functionality.